frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeSidebar.unit.spec.tsx

```diff
@@ -225,26 +215,88 @@ describe("SummarizeSidebar", () => {
   });
 
   it("should list breakouts added before opening the sidebar in a separate section", async () => {
-    await setup({ card: createSummarizedCard() });
-
-    const pinnedColumnList = screen.getByTestId("pinned-dimensions");
-    const unpinnedColumnList = screen.getByTestId("unpinned-dimensions");
+    await setup({ query: createSummarizedQuery() });
 
     expect(
-      within(pinnedColumnList).getByText("Product → Category"),
+      within(getPinnedColumnList()).getByText("Product → Category"),
     ).toBeInTheDocument();
     expect(
-      within(pinnedColumnList).getByText("Created At"),
+      within(getPinnedColumnList()).getByText("Created At"),
     ).toBeInTheDocument();
 
     expect(
-      within(unpinnedColumnList).queryByText("Category"),
+      within(getUnpinnedColumnList()).queryByText("Category"),
     ).not.toBeInTheDocument();
 
     // "Product → Created At" and "User → Created At" should still be there
-    expect(within(unpinnedColumnList).getAllByText("Created At")).toHaveLength(
-      2,
+    expect(
+      within(getUnpinnedColumnList()).getAllByText("Created At"),
+    ).toHaveLength(2);
+  });
+
+  it("should show mulitple breakouts of the same column", async () => {
+    await setup({
+      query: createQueryWithBreakoutsForSameColumn(),
+    });
+    expect(
+      within(getPinnedColumnList()).getAllByText("Created At"),
+    ).toHaveLength(2);
+  });
+
+  it("should allow to modify temporal units for breakouts of the same column", async () => {
```

Comment:
1. Remove the `breakoutPositions.length === 0` as mentioned in PR description
2. New > Question > Orders
3. Add a breakout "Created At: Month"
4. Add another breakout "Created At: Year"
5. Visualize
6. Save the question (if this step is skipped, then breakouts will just disappear in the last step, nothing will crash)
7. Via Summarize sidebar change "Year" to "Month" on the 2nd breakout

Actual: visualization crashes, there are errors in JS console, breakouts disappear
Expected: noop? UI should not allow user to do that?

Reply:
nit: after [recent changes](https://github.com/metabase/metabase/pull/46555#discussion_r1711903238) it no longer crashes but it's not really a noop - pivot table surprisingly gets enabled in the visualization settings (in step 7) even though query does not change